docs/csharp/tutorials/string-interpolation.md

```diff
@@ -0,0 +1,113 @@
+﻿---
+title: String Interpolation | C#
+description: Learn how string interpolation works in C# 6
+keywords: .NET, .NET Core, C#, string
+author: mgroves
+manager: BillWagner
+ms.date: 12/09/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: csharp
+ms.assetid: f8806f6b-3ac7-4ee6-9b3e-c524d5301ae9
+---
+
+# String Interpolation in C#
+
+## Introduction
+
+- string.format
+- other languages
+- how it looks in c# now
+
+## Prerequisites
+You’ll need to setup your machine to run .NET core. You can find the
+installation instructions on the [.NET Core](https://www.microsoft.com/net/core)
+page.
+You can run this application on Windows, Ubuntu Linux, macOS or in a Docker container. 
+You’ll need to install your favorite code editor. The descriptions below
+use [Visual Studio Code](https://code.visualstudio.com/) which is an open
+source, cross platform editor. However, you can use whatever tools you are
+comfortable with.
+
+## Create the Application
+
+Now that you've installed all the tools, create a new .NET Core
+application. To use the command line generator, execute the following command in your favorite shell:
+
+`dotnet new`
+
+This command will create a barebones .NET core project with `project.json` and `Program.cs` files. You will need to execute `dotnet restore` to restore the dependencies needed to compile this project.
+
+To execute the program, use `dotnet run`. You should see "Hello, World" output to the console.
+
+## Intro to String Interpolation
+
+With `String.Format`, you would specify "placeholders" in a string that would be replaced by the parameters following the string. For instance:
```

Comment:
If you use '@System.String.Format' this will become a link to the API reference. (It will display 'String.Format')